contracts/introspection/ERC165Checker.sol

```diff
@@ -0,0 +1,107 @@
+pragma solidity ^0.4.24;
+
+
+/**
+ * @title ERC165Checker
+ * @dev Use `using ERC165Checker for address`; to include this library
+ *      https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md
+ */
+library ERC165Checker {
+  // As per the EIP-165 spec, no interface should ever match 0xffffffff
+  bytes4 private constant InterfaceId_Invalid = 0xffffffff;
+
+  // As per the EIP-165 spec, ERC165_ID == bytes4(keccak256('supportsInterface(bytes4)')
```

Comment:
Could you update this comment to include the '0x01ffc9a7' value? That way there'll be no doubt regarding where it comes from.

Reply:
actually, could we update this to follow the pattern we're using  elsewhere for interface ids?

```
  bytes4 internal constant InterfaceId_ERC721 = 0x80ac58cd;
  /*
   * 0x80ac58cd ===
   *   bytes4(keccak256('balanceOf(address)')) ^
   *   bytes4(keccak256('ownerOf(uint256)')) ^
   *   bytes4(keccak256('approve(address,uint256)')) ^
   *   bytes4(keccak256('getApproved(uint256)')) ^
   *   bytes4(keccak256('setApprovalForAll(address,bool)')) ^
   *   bytes4(keccak256('isApprovedForAll(address,address)')) ^
   *   bytes4(keccak256('transferFrom(address,address,uint256)')) ^
   *   bytes4(keccak256('safeTransferFrom(address,address,uint256)')) ^
   *   bytes4(keccak256('safeTransferFrom(address,address,uint256,bytes)'))
   */
```

Reply:
Ok, pushing a quick fix for this shortly.

Some strangeness/inconsistency here though
1. Why does the comment go *after* the field instead of before it?
2. Some places in this repo ([link1]) use a natspec-style double asterisk comment `/**` even though its after the field and thus not natspec
3. We are using triple equals in a solidity context ([link1] [link2])

[link1]: https://github.com/OpenZeppelin/openzeppelin-solidity/blob/4c6575bf2fdf97af517b9c1aea886125bab8c4af/contracts/introspection/SupportsInterfaceWithLookup.sol#L14
[link2]: https://github.com/OpenZeppelin/openzeppelin-solidity/blob/b120b9aff79d04352778d4e5fd773d987727e1cc/contracts/token/ERC721/ERC721Basic.sol#L14

Reply:
Hmm, agree with you on this. I opened #1182 to address this. Thanks!